node/scripts/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/root/ibm-block-csi-node-driver $@
```

Comment:
What's this for, why you need to call the node driver in this way?

Reply:
To be consistent with the way it is done in controller
Istead of starting in ENTRYPOINT directly added script which does it.

Reply:
first of all, it is not part of this pr, you should do it in other prs.
And you should also use exec as controller did.